pandas/tests/frame/test_api.py

```diff
@@ -365,7 +365,7 @@ def test_to_numpy_copy(self):
         df = pd.DataFrame(arr)
         assert df.values.base is arr
         assert df.to_numpy(copy=False).base is arr
-        assert df.to_numpy(copy=True).base is None
+        assert df.to_numpy(copy=True).base is not arr
```

Comment:
This had to change because now the output array is a view, but only on the transpose of itself (which is a copy of the original data)